apps/shared/function_active_function_toggle.h

```diff
@@ -0,0 +1,27 @@
+#ifndef LAURYUPSILON_FUNCTION_ACTIVE_FUNCTION_TOGGLE_H
```

Comment:
Why do you use this name ?
```suggestion
#ifndef FUNCTION_ACTIVE_FUNCTION_TOGGLE_H
```

Reply:
OH ! It's Clion which generate this name ^^